modules/virt-configuring-vm-disk-sharing.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-vm-disk-sharing{context}"]
+= Configuring disk sharing by using virtual machine disks
+
+You can configure a disk's block volumes to allow multiple virtual machines to share the same storage. 
```

Comment:
Something to check with SME. "disk's block volumes" sounds odd. I think it should be "You can configure block volumes ...".

Reply:
Done